### PR TITLE
add Moyo Header task

### PR DESCRIPTION
**Pull request by TamaraShukalo**
- [DEMO LINK](https://TamaraShukalo.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://TamaraShukalo.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,43 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops-computers" class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
```
Comment (ManoylovAC) [inline src/index.html:30]: please fix your indentations
check code style for html in your [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) (13 point)
if you want to keep attributes on separate line, please keep all of them in that way for current case
```suggestion
            <tag
               attr1
               ...
            >
             Laptops & Computers
           </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,79 @@
...
+  text-transform: uppercase;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
```
Comment (ManoylovAC) [inline src/style.css:55]: please create and use variables for color-values you using several times
```suggestion
  color: var(--some-variable-name);
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,79 @@
...
+  left: 0;
+  right: 0;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
+
+@media screen and (max-width: 1200px) {
+  .header {
+    padding: 0 50px;
```
Comment (ManoylovAC) [inline src/style.css:71]: by requirements you need to work with `width` here
this padding the same as [main rule](https://github.com/mate-academy/layout_moyo-header/pull/3239/files#diff-1104bdfdc4d3b01a85d7af489598121125db6bc016db2a48a083ef654c97b0f5R11) so no need to use it here
image

File: src/style.css [hunk 4]
```diff
@@ -1 +1,79 @@
...
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .header {
+    padding: 0 50px;
```
Comment (ManoylovAC) [inline src/style.css:77]: you need to work with width here as well

**Review by ManoylovAC — CHANGES_REQUESTED** [review]
Hey - good start 👋 
Let's make some improvements by comments I left for you.
Good luck!

File: src/index.html [hunk 5]
```diff
@@ -6,9 +6,48 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#laptops-computers"
+            class="nav__link"
+            data-qa="hover"
+            >
```
Comment (tetiana-ivanchenko) [inline src/index.html:33]: image

File: src/style.css [hunk 6]
```diff
@@ -1 +1,83 @@
...
+  --nav-color-active: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  background: #fff;
```
Comment (tetiana-ivanchenko) [inline src/style.css:11]: it's default value, so here it's redundant

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 7]
```diff
@@ -1 +1,82 @@
...
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  white-space: nowrap;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
```
Comment (YegorKorenkov) [inline src/style.css:47]: Don't forget to add more fallback fonts!

**Review by YegorKorenkov — APPROVED** [review]
Nice work! 🔥 Also don't forget read checklist carefully

**Pull request closed without merging**